### ISSUE TITLE
[BUG] Status field for navigate entity not working

### ISSUE THREAD

**Issue by borky** [joBr99/nspanel-lovelace-ui#379]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._
Grid Card not showing(black screen) when there is configured  a navigate entity with status field. Worked in version 3.1.1.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Provide your apps.yaml:
```
  apps.yaml here:
---
nspanel-birou:
  module: nspanel-lovelace-ui
  class: NsPanelLovelaceUIManager
  config:
    panelRecvTopic: "tele/nsbirou/RESULT"
    panelSendTopic: "cmnd/nsbirou/CustomSend"
    model: us-l
    sleepTimeout: 20
    sleepTracking: device_tracker.pavel
    sleepBrightness:
      - time: "7:00:00"
        value: 10
      - time: "23:00:00"
        value: 1
    sleepOverride:
      entity: light.plafoniera_dormitor_deconz
      brightness: 20
    locale: "ro_RO"
    timeAdditionalTemplate: " Nivel poluare in exterior: {% if is_state('sensor.u_s_air_pollution_level', 'good')-%} bun {% elif is_state('sensor.u_s_air_pollution_level', 'moderate')%} moderat {% else %} nesanatos {% endif -%}"
    # dateAdditionalTemplate: " - {{ states('sun.sun') }}"
    screensaver:
      entity: weather.dark_sky
      alternativeLayout: True
      theme:
        autoWeather: true
      statusIcon1:
        entity: binary_sensor.stare_ac
        icon: mdi:air-conditioner

    hiddenCards:
      - type: cardGrid
        title: Iluminare
        entities:
          - entity: light.plafoniera_dormitor_deconz
            name: Plafoniera
          - entity: light.ambient_dormitor
            name: Ambient
          - entity: light.leduri_noptiere_dormitor
            name: Noptiere
          - entity: input_boolean.master_bedroom_lights_auto_start
            name: Detectie Miscare
        key: iluminare
      - type: cardThermo
        title: AC
        entity: climate.aer_conditionat_living
        key: ac
      - type: cardThermo
        title: Calorifer
        entity: climate.netatmo_dormitor_pavel
        key: caldura

    cards:
      - type: cardGrid
        title: Scurtaturi
        key: scurtaturi
        entities:
          - entity: navigate.cardGrid_iluminare
            name: Iluminare
            icon: mdi:lightbulb-group
          - entity: navigate.cardThermo_ac
            status: binary_sensor.stare_ac # not working anymore, if i remove this line the grid card will work
            name: AC
            icon: mdi:air-conditioner
          - entity: script.turn_on_radio
            name: Radio
            icon: mdi:radio
            # status: media_player.dormitor
            # state_not: playing
      - type: cardGrid
        title: Conditii Camera
        entities:
          - entity: sensor.multi_sensor_bedroom
            name: Temperatura
          - entity: sensor.multi_sensor_bedroom_2
            name: Umiditate
          - entity: sensor.purifier_air_quality
            name: Calitate aer
          - entity: sensor.alimentare_priza_principala_dormitor_mare_power
            name: Consum birou
          - entity: binary_sensor.fum_dormitor_mare
            name: Senzor Fum
      - type: cardMedia
        entity: media_player.dormitor
        # mediaControl: navigate.cardGrid_iluminare
        key: dormitor


```
- [x] Go to Settings>Add-ons>AppDaemon>Log and then, provide the output of the log during your issue/bug occurs:
```
  Log output here:
2022-07-30 10:20:55.600151 INFO nspanel-birou: Received Message from Screen: event,buttonPress2,screensaver,bExit,1
2022-07-30 10:20:55.601597 INFO nspanel-birou: Button Press Event; entity_id: screensaver; button_type: bExit; value: 1 
2022-07-30 10:20:55.603398 INFO nspanel-birou: Started rendering of page 0 with type cardGrid
2022-07-30 10:20:55.605531 INFO nspanel-birou: Sending MQTT Message: pageType~cardGrid
2022-07-30 10:20:55.637101 WARNING nspanel-birou: ------------------------------------------------------------
2022-07-30 10:20:55.637444 WARNING nspanel-birou: Unexpected error in worker for App nspanel-birou:
2022-07-30 10:20:55.637686 WARNING nspanel-birou: Worker Ags: {'id': '91c49f4e6aae4262b1f2de6dbd4a17af', 'name': 'nspanel-birou', 'objectid': 'c58d6797eb984aaa8870ef8e7cb943ac', 'type': 'event', 'event': 'MQTT_MESSAGE', 'function': <bound method LuiMqttListener.mqtt_event_callback of <luibackend.mqttListener.LuiMqttListener object at 0x7f64155dba00>>, 'data': {'topic': 'tele/nsbirou/RESULT', 'wildcard': None, 'payload': '{"CustomRecv":"event,buttonPress2,screensaver,bExit,1"}'}, 'pin_app': True, 'pin_thread': 0, 'kwargs': {'topic': 'tele/nsbirou/RESULT', '__thread_id': 'thread-0'}}
2022-07-30 10:20:55.638018 WARNING nspanel-birou: ------------------------------------------------------------
2022-07-30 10:20:55.638617 WARNING nspanel-birou: Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/appdaemon/threading.py", line 950, in worker
    funcref(args["event"], data, self.AD.events.sanitize_event_kwargs(app, args["kwargs"]))
  File "/config/appdaemon/apps/nspanel-lovelace-ui/luibackend/mqttListener.py", line 58, in mqtt_event_callback
    self._controller.button_press(entity_id, btype, value)
  File "/config/appdaemon/apps/nspanel-lovelace-ui/luibackend/controller.py", line 206, in button_press
    self._pages_gen.render_card(self._current_card)
  File "/config/appdaemon/apps/nspanel-lovelace-ui/luibackend/pages.py", line 511, in render_card
    self.generate_entities_page(navigation, card.title, card.entities, card.cardType)
  File "/config/appdaemon/apps/nspanel-lovelace-ui/luibackend/pages.py", line 300, in generate_entities_page
    command += self.generate_entities_item(item, cardType)
  File "/config/appdaemon/apps/nspanel-lovelace-ui/luibackend/pages.py", line 182, in generate_entities_item
    icon_color = self.get_entity_color(entity, overwrite=colorOverride)
UnboundLocalError: local variable 'entity' referenced before assignment
2022-07-30 10:20:55.638901 WARNING nspanel-birou: ------------------------------------------------------------
2022-07-30 10:21:00.094193 INFO nspanel-birou: Sending MQTT Message: time~10:21~Nivel poluare in exterior:  moderat

```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots/pictures to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

### PANEL / FIRMWARE VERION
_Please add the Panel/Firmware Version you are using (EU, US-L or US-P)_
version 3.2.0 US-L

**Comment by joBr99** [comment]
Hey, should be fixed in v3.2.1

**Comment by borky** [comment]
@joBr99 It was fixed in 3.2.1 as you said, but now in 3.2.2 it appeared again.

**Comment by joBr99** [comment]
lol you are right, no clure where this change went, update to 3.2.3
